spec/bounty_parser_spec.rb

```diff
@@ -0,0 +1,522 @@
+require 'bounty'
+
+class Bounty
+  describe Parser, "#parse" do
+    it "can tell when we don't have a task" do
+      bounty = described_class.parse "You are not currently assigned a task."
```

Comment:
I am not sure this is the best thing to return for this case. Returning a bare `nil` make it hard to tell the different between no bounty assigned and unable to parse a bounty. I have gone back and forth with making task and/or status nil so curious if other people have input on how this example should behave before we release a new API.

Reply:
Updated this to return `:none` based on feedback from Ondreian which feels better.